index.html

```diff
@@ -2,20 +2,210 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Donice Conference</title>
-    <link rel="stylesheet" href="style.css" />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
-        rel="stylesheet" />
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Donice Conference</title>
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+    rel="stylesheet" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&display=swap"
+    rel="stylesheet" />
 </head>
 
 <body>
+  <header>
+    <nav>
+      <div class="nav-mobile hamburger">
+        <div class="bar"></div>
+      </div>
+      <a href="/" class="theme" id="logo">
+        <img src="./assets/shared/logo.svg" alt="logo" />
+        SoloDev
+      </a>
 
+      <ul class="nav-desktop">
+        <li><a href="./about.html" >About</a></li>
+        <li><a href="#program">Program</a></li>
+        <li><a href="#join">Join</a></li>
+        <li><a href="#sponsor">Sponsor</a></li>
+        <li><a href="#news">News</a></li>
+        <li><a href="#campaign" id="campaign">CC Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
```

Comment:
- [ ]  As the design requires we should keep spacesisetween elements, kindly make sure the you use the same thing for your desketop menu items. Check this

Your design | the template
---|---
![image](https://github.com/donice/Microverse-Portal-Capstone/assets/65068771/66bbe240-7f9b-4b03-93fc-dc378ba4f645) | ![image](https://github.com/donice/Microverse-Portal-Capstone/assets/65068771/1e351731-9cd4-4595-94c9-e299cec095b3)

- [ ] I think there is a missing tob navigation bar, kindly add it as required in the design guidlines. 

![image](https://github.com/donice/Microverse-Portal-Capstone/assets/65068771/e897bff8-8ee5-45b1-bc4b-10b3d8f35b7f)